.github/pull_request_template.md

```diff
@@ -3,5 +3,5 @@ Please always provide the [GitHub issue(s)](../issues) your PR is for, as well a
 Fix #<gh-issue-id>
 
 Test URLs:
-- Before: https://main--<repo>--<owner>.hlx.page/
-- After: https://<branch>--<repo>--<owner>.hlx.page/
+- Before: https://main--summit-emea-team-2--adobe-basel.hlx.live/
```

Comment:
Would it make sense to set the default to `.page` as the project is early stage ? Once live, it's more likely to compare with live.

Reply:
You're right!